.github/workflows/contract-testing.yaml

```diff
@@ -69,11 +69,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: setup Atlas CLI
-        uses: andreaangiolillo/atlas-cli-github-action@main
-        with:
-          public-key: ${{ secrets.CLOUD_DEV_PUBLIC_KEY }}
-          private-key: ${{ secrets.CLOUD_DEV_PRIVATE_KEY }}
-          org-id: ${{ secrets.CLOUD_DEV_ORG_ID }}
+        uses: mongodb/atlas-github-action@v0.2.0
```

Comment:
i think we decided to use SHAs in actions

Reply:
Thanks @lantoli great catch, updated to use SHA

Reply:
thx for addressing this